bundles/io/org.eclipse.smarthome.io.transport.bluetooth/OSGI-INF/BluetoothTransportRegistry.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 1997, 2015 by Huawei Technologies Co., Ltd. and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.io.transport.bluetooth.BluetoothTransportProvider">
```

Comment:
The component name here should rather be "org.eclipse.smarthome.io.transport.bluetooth.BluetoothManager", also the filename should be BluetoothManager.xml.